Loadouts/common.hpp

```diff
@@ -29,8 +29,9 @@
 #define BASE_ARTILLERY "ACE_artilleryTable",BASE_BALLISTICS
 
 // OPTIX
-#define WARSAW_OPTICS "rhs_acc_1p63", "rhs_acc_ekp1", "rhs_acc_ekp8_02", "rhs_acc_pkas" // note RHS and CUP mount optics differently, not cross compatible
-#define STANAG_OPTICS "optic_Aco", "optic_Yorris", "rhs_acc_1p87", "rhs_acc_ekp8_18", "rhs_acc_rakursPM", "rhsusf_acc_compm4", "rhsusf_acc_eotech_xps3", "rhsusf_acc_T1_high", "CUP_optic_MRad", "CUP_optic_ZDDot"
+#define WARSAW_OPTICS "CUP_optic_1p63", "rhs_acc_ekp8_02", "CUP_optic_Kobra", "rhs_acc_okp7_dovetail", "rhs_acc_pkas" // RHS and CUP optics now compatible
+#define STANAG_OPTICS "optic_Aco", "ptv_compm4s", "CUP_optic_MicroT1", "ptv_rmr", "CUP_optic_Eotech553_Black", "ptv_xps3", "CUP_optic_ZDDot", "CUP_optic_MRad"
+#define BAD_OPTICS "rhs_acc_rakursPM", "rhs_acc_1p87", "CUP_optic_AC11704_Black", "CUP_optic_MARS", "CUP_optic_MEPRO", "CUP_optic_TrijiconRx01_black", "CUP_optic_ZeissZPoint" //These are alternate optics that kinda suck, but are unique
```

Comment:
this won’t be valid in loadout; the comment at the end will cause the array to not close. replace with `/* These are alternate optics that kinda suck, but are unique */`

Reply:
```suggestion
//These are alternate optics that kinda suck, but are unique
#define BAD_OPTICS "rhs_acc_rakursPM", "rhs_acc_1p87", "CUP_optic_AC11704_Black", "CUP_optic_MARS", "CUP_optic_MEPRO", "CUP_optic_TrijiconRx01_black", "CUP_optic_ZeissZPoint"
```